extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -164,25 +187,24 @@ export class AmpSlideScroll extends BaseSlides {
 
   /** @override */
   onLayoutMeasure() {
-    /** @private {number} */
     this.slideWidth_ = this.getLayoutWidth();
 
-    /** @private {number} */
     this.previousScrollLeft_ = this.slidesContainer_./*OK*/scrollLeft;
   }
 
   /** @override */
   layoutCallback() {
-    if (this.slideIndex_ == null) {
+    if (this.slideIndex_ === null) {
```

Comment:
nvm, its just the normal devirtualization/extractPrototypeMemeberDeclaration optimizations its doing to methods. first time seeing it this closely